Library/Homebrew/utils/bottles.rb

```diff
@@ -74,6 +74,77 @@ def formula_contents(bottle_file,
 
         contents
       end
+
+      def add_bottle_stanza!(formula_contents, bottle_output)
+        require "rubocop-ast"
```

Comment:
```suggestion
        Homebrew.install_bundler_gems!
        require "rubocop-ast"
```

Not technically necessary but semantically it might also make sense to add `rubocop-ast` to the `Gemfile` directly now that we're using it directly (for example in case RuboCop ever stopped depending on it).